src/entryPoints/HyperSwitchApp.res

```diff
@@ -236,6 +237,9 @@ let make = () => {
                         }}
                       />
                     </div>
+                    <RenderIf condition=isLiveUsersCounterEnabled>
```

Comment:
in which environment are we going to enable this feature

Reply:
It will be supported in localhost as well as sandbox and production